kernel/context/mod.rs

```diff
@@ -26,7 +26,7 @@ pub mod file;
 pub mod memory;
 
 /// Limit on number of contexts
-pub const CONTEXT_MAX_CONTEXTS: usize = 65536;
+pub const CONTEXT_MAX_CONTEXTS: usize = usize::max_value();;
```

Comment:
Extra semicolon. Also, I think it should be usize::max_value() - 1 so that it doesn't cause an overflow panic